ui/src/app/view/timestamp-table/timestamp-table.component.scss

```diff
@@ -20,6 +20,13 @@
   }
 }
 
+tr{
+  height: 59px !important;
+}
+
+.invisable{
+  height:1px!important;
+}
```

Comment:
Let us try to avoid using `!important`

Reply:
Should probably also be 'invisible' instead of 'invisable'

Reply:
> Should probably also be `invisible` instead of `invisable`

The typo was consistently used in the code (so if we change it here, we should change it else where too)